actiontext/lib/action_text/fixture_set.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module ActionText
+  class FixtureSet
+    def self.attachment(fixture_set_name, label, column_type: :integer)
+      signed_global_id = ActiveRecord::FixtureSet.signed_global_id fixture_set_name, label,
```

Comment:
@georgeclaghorn do you think that this is a reasonable addition to the ActiveRecord side of fixtures, or would we be better keeping it self-contained to the ActionText side?